service/src/test/java/bio/terra/workspace/service/workspace/WorkspaceUnitTest.java

```diff
@@ -24,4 +28,19 @@ void workspaceRequiredFields() {
         MissingRequiredFieldsException.class,
         () -> Workspace.builder().workspaceStage(WorkspaceStage.MC_WORKSPACE).build());
   }
+
+  @Test
+  void cloudContextSerDes() throws Exception {
+    ObjectMapper objectMapper = StairwayMapper.getObjectMapper();
+    GcpCloudContext serContext = new GcpCloudContext("terra-wsm-t-potent-cashew-2078");
+    String serJson = objectMapper.writeValueAsString(serContext);
+    String json =
+        "[\"bio.terra.workspace.service.workspace.model.GcpCloudContext\",{\"gcpProjectId\":\"terra-wsm-t-potent-cashew-2078\"}]";
+    if (StringUtils.equals(serJson, json)) {
```

Comment:
why not use `AssertEquals`?

Reply:
That test was some debug code. I removed it.